tools/roslaunch/scripts/roscore

```diff
@@ -49,6 +49,12 @@ def _get_optparse():
     parser.add_option("-v", action="store_true",
                       dest="verbose", default=False,
                       help="verbose printing")
+    parser.add_option("-w", "--numworkers",
+                      dest="nwork", default=3, type="int",
+                      help="override number of worker threads", metavar="NUM_WORKERS")
+    parser.add_option("-t", "--timeout",
+                      dest="timeout", default=-1.0, type="float",
```

Comment:
I would suggest no default here which implies `None` and handle that accordingly.